pkg/kubelet/kubelet.go

```diff
@@ -387,7 +387,7 @@ func NewMainKubelet(
 			return nil, err
 		}
 		klet.containerRuntime = rktRuntime
-		klet.pleg = pleg.NewGenericPLEG(klet.containerRuntime, plegChannelCapacity, plegRelistPeriod)
+		klet.pleg = pleg.NewGenericPLEG(klet.containerRuntime, plegChannelCapacity, plegRelistPeriod, nil)
```

Comment:
[This](https://github.com/kubernetes/kubernetes/pull/17700#discussion_r49015593) comment get lost
> This line is the same as line 373, can you move it outside of the switch-case ?